tests/utils.go

```diff
@@ -456,6 +448,8 @@ func BeforeTestSuitSetup() {
 	log.InitializeLogging("tests")
 	log.Log.SetIOWriter(GinkgoWriter)
 
+	Config = loadConfig()
```

Comment:
can we move this into the init block of config.go?

Reply:
Unfortunately it does not work since the command line flags are not parsed yet in `init()`.

Reply:
ok, too bad